lib/semian.rb

```diff
@@ -159,6 +160,10 @@ def resources
 require 'semian/simple_sliding_window'
 require 'semian/simple_integer'
 require 'semian/simple_state'
+require 'semian/sysv_shared_memory'
```

Comment:
Should these `require`s be conditional depending on whether the target platform supports SysV?